app/controllers/questionnaire_controller.rb

```diff
@@ -25,7 +25,9 @@ def create_informed_consent
 
   def create
     response_content = ResponseContent.create!(content: questionnaire_content)
-    @response.update_attributes!(content: response_content.id, completed_at: Time.zone.now)
+    @response.update_attributes!(content: response_content.id,
+                                 completed_at: Time.zone.now,
+                                 filled_out_for: @prototocol_subscription.filling_out_for)
```

Comment:
Moet je hier niet ook direct de `filled_out_by` property opslaan? Of doe je dat ergens anders.